src/codemodder/registry.py

```diff
@@ -152,7 +152,7 @@ def match_codemods(
 def load_registered_codemods() -> CodemodRegistry:
     registry = CodemodRegistry()
     logger.info("Loading registered codemod collections")
-    for entry_point in entry_points()["codemods"]:
+    for entry_point in entry_points().select(group="codemods"):
```

Comment:
Fixed a DeprecationWarning. See https://importlib-metadata.readthedocs.io/en/latest/using.html#entry-points

Reply:
Not sure whether you ran into this problem with the change but I think the right way to fix this is probably by adding some conditional logic based on Python version.

Reply:
I ended up removing this change (sorry forgot to remove my comment) bc yeah py versions. We can leave as is for now.